src/commcare_cloud/environment/secrets/secrets.yml

```diff
@@ -177,3 +177,5 @@
   legacy_namespace: secrets
 - name: ABDM_CLIENT_SECRET
   legacy_namespace: secrets
+- name: FCM_CREDS
+  legacy_namespace: secrets
```

Comment:
Sure will make this change.

Reply:
Hey @millerdev 
After making this change (removing the namespace), I see an error in the workflow run. 
`An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleError: An unhandled exception occurred while templating '{{ FCM_CREDS | default(None) }}'`

Details here: https://github.com/dimagi/commcare-cloud/actions/runs/5190733656/jobs/9375718229#step:5:737

Let me know in case of any inputs.
I am planning to revert this and get this going as would like to deploy on staging.

Reply:
It's interesting that the error ends with a slightly different error:
```
original message: recursive loop detected in template string: {{ FCM_CREDS | default(None) }}
```
I'm not sure what it means. It appears that `slack_token`, which appears three lines above the only occurrence of that template string, is using the same technique. Maybe the difference is that `slack_token` is never referenced in a jinja2 template file?

I don't know what's going on. Fine to revert.

Reply:
Yes , `slack_token` is not referenced in Jinja template.